test/perf/perf_simple_query.cc

```diff
@@ -603,19 +604,35 @@ int scylla_simple_query_main(int argc, char** argv) {
                     : do_alternator_test(app.configuration()["alternator"].as<std::string>(),
                             env.local_client_state(), env.qp(), env.migration_manager(), env.gossiper(), cfg);
 
-            auto compare_throughput = [] (perf_result a, perf_result b) { return a.throughput < b.throughput; };
-            std::sort(results.begin(), results.end(), compare_throughput);
+            std::string stats_metric = "throughput";
+            if (app.configuration().contains("stats-metric")) {
+                stats_metric = app.configuration()["stats-metric"].as<std::string>();
+            }
+            std::function<double(const perf_result& r)> get_metric;
```

Comment:
could you do it in a way that perf_alternator also benefits? and maybe perf_commitlog ? 

It would be useful for me in perf_alternator at least.